entrypoint.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 
 set -o pipefail
+npm install -g semver
```

Comment:
Is there any specific reason/preference for installing `semver` here vs. the docker image build?

Reply:
it bumps the prereleases. It was never implemented in the `semserver-tool`, see [here](https://github.com/fsaintjacques/semver-tool/issues/25)

Reply:
Right, I'm happy to have this installed instead of copying over the other script, but perhaps it is better to have this installed in the Dockerfile?

Reply:
I’ve never used a _Dockerfile_ before and I’m not sure if it’s advantages to be honest nor what can you do in it. Can I just add `... && npm install -g semver`?

Reply:
pushed new change to move the installation to the dockerfile

Reply:
The only real benefit would be if Github cached/saved the built docker image from one action run to another, which I'm not sure that they do. But thanks for changing it 👍